documentation/docs/core-concepts/hello-world.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Hello World
+path: /hello-world/
+---
+
+# Hello World
+
+A simple React app using Fusion.js would look something like this:
+
+```js
```

Comment:
Maybe it would be helpful to mention that this would be in the `src/main.js` file. If someone doesn't know to put it in this file and run `fusion dev`, they may be a bit confused how this is run. I'm not sure if you intentionally left this out to keep it more clean and with the expectation that people would not try to run it.

Reply:
Yeah the idea was that people wouldn't necessarily run this code, just to give them a code snippet to look at to get an example of what a Fusion app looks like.

Reply:
Ok, seems fine, though that is what I did when I saw this code :)